### PR TITLE
Added support for Wiren Board MQTT Conventions

### PR DESCRIPTION
**Pull request by aqualx**
Added support for [Wiren Board](https://wirenboard.com/en/) controller that uses [homeA-like MQTT conventions](https://github.com/binarybucks/homA/wiki/Conventions). This conventions are published [here](https://github.com/contactless/homeui/blob/master/conventions.md)

### CODE REVIEW

File: miflora-mqtt-daemon.py [hunk 1]
```diff
@@ -303,6 +305,22 @@ def flores_to_openhab_items(flores, reporting_mode):
             if 'device_class' in params:
                 payload['device_class'] = params['device_class']
             mqtt_client.publish('{}/{}_{}/config'.format(topic_path, flora_name, sensor).lower(), json.dumps(payload), 1, True)
+elif reporting_mode == 'wirenboard-mqtt':
```
Comment (ThomDietrich) [inline miflora-mqtt-daemon.py]: Seems like the `base_topic` is not regarded. Could you please print an informative warning here, if one is defined!? Thanks

**Comment by ThomDietrich** [comment]
I did not expect so many conventions when first developing this tool 😄 I'll soon switch to a class based structure for this. Would it be okay for you to check functionality then?

Your PR looks good to me. Thanks!

**Comment by aqualx** [comment]
> I did not expect so many conventions when first developing this tool 😄 I'll soon switch to a class based structure for this. Would it be okay for you to check functionality then?
> 
> Your PR looks good to me. Thanks!

Of course it will not be a problem to test for me!

Thanks for nice daemon!

File: miflora-mqtt-daemon.py (hunk 1 above)
Comment (ThomDietrich) [reply miflora-mqtt-daemon.py]: Just to make sure: did you see this comment?

Comment (aqualx) [reply miflora-mqtt-daemon.py]: Yes, I saw. I will add warning today.

**Comment by aqualx** [comment]
> I'll soon switch to a class based structure for this.

Maybe it is also good idea to implement support of other types of BLE sensors?
Seems that this not a problem to add support for "Xiaomi Mi Temperature and Humidity Sensor" using library [mitemp](https://github.com/ratcashdev/mitemp).
I was almost going to extend your project with such support... But now, probably, I should wait till the end of your refactoring.

**Comment by ThomDietrich** [comment]
Valuable point. And yes I agree. Let's first refactor the code base. There are many things I've planned for the next version. Addition of other sensors will be much easier after that.

**Pull request merged**